content/blog/free-resources-for-cloud-engineering/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Guide to Free Resources for Cloud Engineering"
+date: 2020-11-12
+meta_desc: "Free resources for building infrastructure with cloud engineering."
+meta_image: cloud_engineering_resources.png
+authors:
+    - sophia-parafina
+tags:
+    - cloud engineering
+    - AWS
+    - Azure
+    - Google Cloud
+---
+
+Innovation is at the heart of cloud engineering. Finding the right cloud solution requires experimentation to understand each cloud ecosystem and the best practices for each. The good news is that cloud providers have free service tiers for many cloud resources. These resources are often always free under a certain usage threshold or free to use for a limited time, often 12 months after creating an account.
+
+Having free access to resources frees you to experiment, try out new ideas, and implement different application deployment architectures. However, given the number of cloud providers and the range of services they provide, getting started can be intimidating. This article is a guide to popular cloud resources and how to use them.
+
+<!--more-->
+
+## AWS
+
+Amazon offers a free tier composed of always free, 12 months free available to new customers, and short term trials. The following is a selection of always free resources with accompanying tutorials, examples, and blog posts to get you started.
```

Comment:
I think we should link to their entire set of free tier docs - https://aws.amazon.com/free/

Reply:
Good suggestion, I'll update this for each provider, it's kinda there for Azure but not obvious